### PR TITLE
[CUBRIDQA-60] add jdbc test into ctp

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

I add jdbc unit test into ctp, please review.

### CODE REVIEW

**Comment by fanzaiqiang** [comment]
@supler 

Great. Please go ahead to merge.  :blush:

**Comment by supler** [comment]
@fanzaiqiang 

add run.sh and configuration file into ctp

File: CTP/jdbc/bin/run.sh [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+function do_init()
+{
+	curDir=`pwd`
+	db_name="jdbcdb"
+	db_charset="en_us"
+	cd ${CTP_HOME}/conf
+    rm runtime_jdbc.conf 2>&1 >/dev/null
+    touch runtime_jdbc.conf 
+    runtime_configuration_file=${CTP_HOME}/conf/runtime_jdbc.conf
```
Comment (fanzaiqiang) [inline CTP/jdbc/bin/run.sh]: Do we require file runtime_jdbc.conf?

File: CTP/jdbc/bin/run.sh [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+        ini -s "%query_editor" -u "${cubrid_broker_conf_queryeditor_para}" $CUBRID/conf/cubrid_broker.conf
+     fi
+
+     if [ "$cubrid_broker_conf_para" ] && [ $is_valid_section_broker1 -ne 0 ];then
+        ini -s "%BROKER1" -u "${cubrid_broker_conf_para}" $CUBRID/conf/cubrid_broker.conf
+     fi
+     
+
+     #make_locale
+     make_locale
```
Comment (fanzaiqiang) [inline CTP/jdbc/bin/run.sh]: It's better to remove make_locale as default.

**Comment by supler** [comment]
@fanzaiqiang 

remove make_locale and configuration file backup

**Pull request merged**